azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -124,7 +124,7 @@ private void loadPluginJobTypes(final JobTypePluginSet plugins)
     } else {
       logger.info("Common plugin job props file " + commonJobPropsFile
           + " not found. Using empty props.");
-      commonPluginJobProps = new Props();
+      commonPluginJobProps = new Props(globalProperties);
```

Comment:
Makes sense to me! Can you call it `this.globalProperties` though so it's in line with our style guide? After that I will ship it + merge 👍 

Thanks for the contribution!